### DISCUSSION TITLE
Hoping for Z-Wave Long Range LR Support

### DISCUSSION THREAD

**Discussion by TheCodeJanitor-dotcom** [zwave-js/zwave-js#6518] [Feature Requests]
### Description

Looking for LR support to replace a Chamberlain garage door solution. 

### Additional context

The Zooz 800 LR stick was on major sale, couldn't resist, and bought a compatible LR open/close sensor for testing.
I replaced my Zooz 700 stick without major issues, and it is working well, but hoping to be able to try out LR application.

Thanks!

**Comment by AlCalzone** [comment]
It's a work in progress: https://github.com/zwave-js/node-zwave-js/pull/6401
